frontend/public/components/catalog/_catalog.scss

```diff
@@ -171,18 +183,47 @@ $co-modal-ignore-warning-icon-width: 30px;
     }
   }
 
+  &__overlay--right {
+    position: fixed !important;
+    bottom: 0;
+    padding: 0px !important;
+    right: 0;
+    top: 76px;
+
+    .catalog-item-header-pf {
+      padding: 24px 24px 16px 24px;
+    }
+
+    .pf-c-modal-box__body {
+      margin: 0 !important;
+    }
+
+    .properties-side-panel-pf-property {
+      width: 165px;
```

Comment:
Since we're already overriding this width, is there any reason we shouldn't just increase this value so that `Instantiate Template` fits and we maintain the existing button placement?

Reply:
@maryshak1996 thought her approach would be more scalable. @bmignano and @maryshak1996, can you chime in?

Reply:
Digging deeper, I see we set a width on `properties-side-panel-pf` upstream, then override that width on 182 (which is currently doing nothing because the SCSS is invalid), to then set the same width here.  Let's not do that.   If we need to override the width, it should be done on 182 once that SCSS is fixed.

Reply:
@rhamilto 
hey so I'm just catching up here -- it looks like we've got two options here (correct me if I'm misunderstanding for sure!):
- Expand the width of the drawer to allow for the button to stay in the main drawer content section (where it originally was) 
- Place the button on its own line and keep the drawer width the same
I'd be in favor of giving the button it's own line, rather than making the drawer wider. The drawer content section is quite tight to begin with due to the 2-column structure, and I think that forcing the button to fit in a narrow column could be problematic in the event of button text needing to be longer than what we have today. It's more scalable to bring the button to the top, and it also creates better hierarchy in the drawer itself.

Reply:
Moving the button to the header is fine with me.   But the implementation should be fixed to restore the inner scroll shadows on the modal.

Reply:
And if the button is going to live in the header, it seems like the button should not scroll.